.github/workflows/dispatch-build-bottle.yml

```
@@ -145,19 +145,19 @@ jobs:
         env:
           HOMEBREW_BINTRAY_USER: brewtestbot
           HOMEBREW_BINTRAY_KEY: ${{secrets.HOMEBREW_BINTRAY_KEY}}
-          HOMEBREW_GPG_PASSPHRASE: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY_PASSPHRASE }}
-          BREWTESTBOT_NAME_EMAIL: "BrewTestBot <1589480+BrewTestBot@users.noreply.github.com>"
         run: |
           cd ~/bottles
-          brew pr-upload --verbose --keep-old --committer="$BREWTESTBOT_NAME_EMAIL"
+          brew pr-upload --verbose --keep-old --no-commit --root-url="https://ghcr.io/v2/homebrew/core"
 
       - name: Upload bottles to GitHub Packages
```

Update the names to match:

```suggestion
      - name: Upload bottles to Bintray
```